packages/tools/generator-widget/generators/app/lib/prompttexts.js

```diff
@@ -6,7 +6,7 @@ function promptWidgetProperties(mxProjectDir, widgetName) {
             type: "input",
             name: "name",
             validate: input => {
-                if (/^([a-zA-Z]+)$/.test(input)) {
+                if (/^([a-zA-Z-]+)$/.test(input)) {
```

Comment:
Why do we support dashes now? The description below doesn't mention this, nor does it seem relevant to this MR.

Reply:
IIUC, I should discard this change as it not relevant to this bugfix?

Reply:
Unless you can give me a very good reason why this change is necessary, yes ;).